library/src/morpheus-master/sass/modules/navigation/_skipnav.scss

```diff
@@ -93,6 +93,17 @@
 				
 				a.#{$namespace}skipNav--toolName {
 					color: $link-color;
+				}
+				a.manage-overview-link-mobile {
+					color: $link-color;
+					cursor: pointer;
+					display: inline-block;
+					font-family: "Open Sans", sans-serif;
```

Comment:
Can these font family declarations be pulled from the default font-family variable in SASS? Are they not cascading from parent elements anyways?